apps/espruinoctrl/custom.html

```diff
@@ -164,6 +164,7 @@
   if (mac) {
     E.showMessage("Connecting\\n"+mac);
     if (mac.length==17) mac+=" random";
+    NRF.setTxPower(process.env.HWVERSION == 2 ? 8 : 4);
```

Comment:
A MAX_POWER constant might be helpful.
also a way to read current NRF settings.

Is there any additional thing to to to increase
range?

Reply:
I guess it would be an idea - or maybe the function could just clip the TX power to the level it can do rather than erroring.

I don't think there is much extra to increase range when connecting. For the peripheral device if it's NRF52840 I think it can now specify to enable long range advertising though